introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi there!
-My favourite Code School path is JavaScript.
\ No newline at end of file
+My favourite Code School path is JavaScript.
+*Csharp
+*Git
+*AngularJS
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?